msteams-platform/toolkit/create-new-project.md

```diff
@@ -8,7 +8,7 @@ ms.topic: overview
 ms.date: 03/14/2022
 ---
 
-# Create a new Teams app using Teams Toolkit
+# Create a new Teams app using Teams Toolkit 
```

Comment:
Why add an extra space at the end of a few titles? It serves no purpose and leads to a markdownlint warning.

Reply:
Got it. My mistake here.

Reply:
Removed the extra space.